design-system/materials/internals/definition.yaml

```diff
@@ -409,6 +409,19 @@ decisionGroupsByTheme:
     # We don't have font families in choices yet, so we can't have decisions for
     # them
 
+    borderWidth:
+      label: Border width
+      prefix: border-width
+      decisions:
+        border-width-for-input:
+          choice: '1px'
```

Comment:
Perhaps we can have new choice tokens with `border-widths` similar to what we already have with `border-radius`.

Reply:
Updated here [a6465b9](https://github.com/commercetools/ui-kit/pull/2347/commits/a6465b999def586cb373a8148e735873420465ed)